CHANGELOG.md

```diff
@@ -28,6 +28,12 @@ The general format is:
 - C from D
 
 ```
+# 1.0.1 - DATE (11/01/2022)
```

Comment:
Hi @bayerasif 

`1.0.1` was already published, so you can rebase main, and change that to `1.0.2`

Reply:
Since this PR aims to add functionality in a backwards compatible manner (new feature), and Hera follows semantic versioning, I suggest we bump to `1.1.0` rather than `1.0.2`. Using `1.0.2` would indicate a patch/bug fix, which is not the aim of the PR. https://semver.org/ is a great resource for more information! 🙂